scripts/gen_xcframework.sh

```diff
@@ -27,14 +27,18 @@ fi
 readonly FRAMEWORK_DIR=$1
 shift
 
-# xcodebuild won't override any existing framework files.
-rm -rf ${FRAMEWORK_DIR}/*
+readonly FRAMEWORK_HEADERS_DIR="${FRAMEWORK_DIR}/Headers"
+
+if [ ! -d "${FRAMEWORK_HEADERS_DIR}" ]; then
+  echo "$FRAMEWORK_HEADERS_DIR should exist but doesn't."
+fi
 
 CMD="xcodebuild -create-xcframework -output "${FRAMEWORK_DIR}
 
 while test ${#} -gt 0
 do
   CMD=${CMD}" -library "$1
+  CMD=${CMD}" -headers "$FRAMEWORK_HEADERS_DIR
```

Comment:
See the [Generate the XCFramework bundle](https://developer.apple.com/documentation/xcode/creating-a-multi-platform-binary-framework-bundle#Generate-the-XCFramework-bundle) Apple Developer guide for an explanation of the `-headers` option.